src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+  
+.files_name {
+  display: none;
+}
+  
+.logo {
+  height: 40px;
+  width: 40px;
+}
+  
+.container {
+  display: flex;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+}
+  
+.nav_list {
+  margin: 0;
+  font-size: 0;
+  padding: 0;
+}
+  
+.nav_item {
+  display: inline-block;
+  text-transform: uppercase;
+  margin-right: 20px;
+  padding: 22.5px 0;
```

```suggestion
  padding: 23px 0;
```
use only integers for px values